docs/topics/proc_attaching-a-remote-debugger-to-the-application.adoc

```diff
@@ -0,0 +1,33 @@
+
+[#attaching-a-remote-debugger-to-the-application]
+= Attaching a Remote Debugger to the Application
+
+When your application is configured for debugging, attach a remote debugger of your choice to it.
+In this guide, The xref:https://www.redhat.com/en/technologies/jboss-middleware/developer-studio[Red Hat JBoss Developer Studio] is covered, but the procedure is similar when using other programs.
```

Comment:
The -> the

Reply:
also, `The Red Hat JBoss Developer Studio is covered` sounds off.

Consider either of the following:
* The Red Hat JBoss Developer Studio debugger is covered
* Red Hat JBoss Developer Studio is covered

Reply:
Fixed